lib/ci/ci_result_parser.js

```diff
@@ -804,7 +839,9 @@ class FannedBuild extends Job {
     const { jobName, buildNumber } = failedPhase;
     const build = new NormalBuild(cli, request, jobName, buildNumber);
     let failures = await build.getResults();
-    failures = flatten(failures);
+    if (failures != undefined){
```

Comment:
Would it be better to change .getResults(), so that it always returns a results array, even if its empty?